src/AutoPlay.ts

```diff
@@ -98,7 +105,7 @@ class AutoPlay implements Plugin {
     // DO-NOTHING
   }
 
-  public play = () => {
+  public play = (duration?: number) => {
```

Comment:
You have to double-think about adding a new parameter, and here's why:

You had to change the event handler from `play` to `_move`, right?
And that was probably because of the type mismatch.
```diff
-  flicking.off(EVENTS.MOVE_END, this.play);
-  flicking.off(EVENTS.SELECT, this.play);
+  flicking.off(EVENTS.MOVE_END, this._move);
+  flicking.off(EVENTS.SELECT, this._move);
```

Bad thing is, the same thing can be applied to any other codebase with the same usage.
So please merge after checking whether the new parameter is really needed, there's no harm in being careful.

Reply:
Keeping this in mind, the behavior of the existing `play` method has been preserved.
Since I changed it to not affect any other codebases, I would like to request a review about it.